consul/compose.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Let's be consistent and self contained: move this logic in the python code and kill the shell script

Reply:
This was for testing, wasn't ever meant to be committed